components/observatorium.libsonnet

```diff
@@ -213,16 +213,23 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       namespace: obs.config.namespace,
       replicas: 1,
       commonLabels+:: obs.config.commonLabels,
-      readEndpoint: 'http://%s.%s.svc.cluster.local:%d' % [
-        obs.queryCache.service.metadata.name,
-        obs.queryCache.service.metadata.namespace,
-        obs.queryCache.service.spec.ports[0].port,
-      ],
-      writeEndpoint: 'http://%s.%s.svc.cluster.local:%d' % [
-        obs.receiveService.metadata.name,
-        obs.receiveService.metadata.namespace,
-        obs.receiveService.spec.ports[2].port,
-      ],
+      logs: {
+        // Fake logs endpoints to satisfy Observatorium flag parsing.
```

Comment:
shouldn't you make these optional instead?

Reply:
They are required today. We can make it optional in a follow up PR in the API repo and then remove these from here in another PR

Reply:
yes this is what I meant to make them optional in the API, all good if you want this in a separate PR.